networking_ccloud/common/config/config_driver.py

```diff
@@ -196,20 +196,42 @@ def check_port_name_in_lacp_mode(cls, values):
 
 class InfraNetwork(pydantic.BaseModel):
     name: str
-    native_vlan_pool: str = None
-    vlan: pydantic.conint(gt=0, lt=4096)
-    networks: List[str] = None
-    vni: pydantic.conint(gt=0, lt=2**24) = None
+    vlan: pydantic.conint(gt=1, lt=4095)
+    vrf: str = None
```

Comment:
Now that you have rebased onto the VRF config branch - do we want to have a check in the driver config that all VRFs referenced by an InfraNetwork exists in our config? Would be an extra root validator on the DriverConfig class

Reply:
Yes, there is a commit doing exactly that.